services/app/lib/codebattle_web/channels/users_channel.ex

```diff
@@ -0,0 +1,21 @@
+defmodule CodebattleWeb.UsersChannel do
+  @moduledoc false
+  use CodebattleWeb, :channel
+
+  alias Codebattle.User.Stats
+
+  require Logger
+
+  def join("users", _payload, socket) do
+    {:ok, %{}, socket}
+  end
+
+  def handle_in("get_info", payload, socket) do
+    %{"id" => id} = payload
+
+    stats = Codebattle.User.Stats.for_user(id)
+
+    broadcast!(socket, "new:user_stats", %{stats: stats, user_id: id})
+    {:noreply, socket}
+  end
+end
```

Comment:
I am not sure but maybe it would be better to put stats to User data shape and get rid of passing userStat to to a bunch of react components.

Reply:
Sorry for my English. I think it will be better to make rest API. Without channels.  And use in react widget. When user hover on any player name we make axios request to fetch data only this user and store it in redux.